src/scripts/downloadSprites.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Manually trigger a redownload of all sprite data
+ */
+const danceParty = require('@code-dot-org/dance-party');
+const SPRITE_NAMES = danceParty.constants.SPRITE_NAMES;
+const MOVE_NAMES = danceParty.constants.MOVE_NAMES;
+const IMAGE_S3_BASE = "http://s3.amazonaws.com/cdo-curriculum/images/sprites/spritesheet_tp/";
```

Comment:
@cpirich _just_ changed this to `spritesheet_tp2`, and I've got another asset layout change coming in real soon (fewer spritesheets per sprite).  Can you help me fix up this process at that point?

Reply:
I'd love to be involved in that process!